src/Security/Security.php

```diff
@@ -206,15 +212,113 @@ class Security extends Controller implements TemplateGlobalProvider
      * @var boolean If set to TRUE or FALSE, {@link database_is_ready()}
      * will always return FALSE. Used for unit testing.
      */
-    static $force_database_is_ready = null;
+    protected static $force_database_is_ready = null;
 
     /**
      * When the database has once been verified as ready, it will not do the
      * checks again.
      *
      * @var bool
      */
-    static $database_is_ready = false;
+    protected static $database_is_ready = false;
+
+    /**
+     * @var array available authenticators
+     */
+    protected static $authenticators = [];
+
+    /**
+     * @var string Default authenticator
+     */
+    protected static $default_authenticator = MemberAuthenticator\Authenticator::class;
```

Comment:
Actually, default_authenticator isn't even used anywhere. Let's remove it entirely.